composer.json

```diff
@@ -10,12 +10,14 @@
         "guzzlehttp/psr7": "~1.4",
         "psr/http-message": "~1.0",
         "psr/cache": "^1.0",
-        "symfony/cache": "*"
+        "symfony/filesystem": "~2.7|~3.0|~4.0",
+        "cache/void-adapter": "^1.0"
     },
     "require-dev": {
         "php-vcr/phpunit-testlistener-vcr": "^2.0",
         "jakub-onderka/php-parallel-lint": "^0.9.2",
-        "jakub-onderka/php-console-highlighter": "^0.3.2"
+        "jakub-onderka/php-console-highlighter": "^0.3.2",
+        "cache/array-adapter": "^1.0"
     },
     "autoload": {
         "files": ["src/functions.php"],
```

Comment:
Can you add a `suggest` block [like this](https://github.com/contentful/contentful-management.php/blob/master/composer.json#L36-L38) to suggest using `symfony/cache`?